src/r2mm/manager/SettingsDexieStore.ts

```diff
@@ -31,7 +31,7 @@ export default class SettingsDexieStore extends Dexie {
         // This contains the following tables:
         // - `games`: (settingsIdentifier,settings) <-- Game-specific settings.
         // - `value`: (++id, settings)              <-- Global settings.
-        this.version(75).stores(store).upgrade(async (tx) => {
+        this.version(76).stores(store).upgrade(async (tx) => {
```

Comment:
Schema version bump is no longer needed for basic game additions, so this should be removed.